account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -35,6 +35,12 @@ def _get_states(self):
                 ('expired', 'Expired'),
                 ('cancel', 'Cancelled')]
 
+    format = fields.Selection(

```

Comment:
Only is to have mandates separated by format, basic and SEPA and other new formats.